aurora-mysql-plugin/src/test/java/co/cask/aurora/mysql/AuroraMysqlPluginTestBase.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.aurora.mysql;
+
+import co.cask.ConnectionConfig;
+import co.cask.DBRecord;
+import co.cask.auroradb.mysql.AuroraMysqlAction;
+import co.cask.auroradb.mysql.AuroraMysqlConstants;
+import co.cask.auroradb.mysql.AuroraMysqlPostAction;
+import co.cask.auroradb.mysql.AuroraMysqlSink;
+import co.cask.auroradb.mysql.AuroraMysqlSource;
+import co.cask.cdap.api.artifact.ArtifactSummary;
+import co.cask.cdap.api.plugin.PluginClass;
+import co.cask.cdap.datapipeline.DataPipelineApp;
+import co.cask.cdap.proto.id.ArtifactId;
+import co.cask.cdap.proto.id.NamespaceId;
+import co.cask.cdap.test.TestConfiguration;
+import co.cask.db.batch.DatabasePluginTestBase;
+import co.cask.db.batch.sink.ETLDBOutputFormat;
+import co.cask.db.batch.source.DataDrivenETLDBInputFormat;
+import com.google.common.base.Charsets;
+import com.google.common.base.Throwables;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Sets;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Map;
+import java.util.TimeZone;
+import javax.sql.rowset.serial.SerialBlob;
+
+public class AuroraMysqlPluginTestBase extends DatabasePluginTestBase {
+  protected static final ArtifactId DATAPIPELINE_ARTIFACT_ID = NamespaceId.DEFAULT.artifact("data-pipeline", "3.2.0");
+  protected static final ArtifactSummary DATAPIPELINE_ARTIFACT = new ArtifactSummary("data-pipeline", "3.2.0");
+  protected static final long CURRENT_TS = System.currentTimeMillis();
```

Comment:
Will using the current time ever cause issues/flakiness for the test? If it will, a hardcoded value may be better.

Reply:
These tests don't rely on any particular timestamp value and are not affected by a timezone change so everything should be fine.